ckanext/datagovtheme/templates/footer.html

```diff
@@ -1,37 +1,213 @@
+              >Office of the Inspector General</a
+            >
+            <svg class="usa-icon" aria-hidden="true" role="img">
+              <use xlink:href="#svg-launch"></use>
+              <symbol viewBox="0 0 24 24" id="svg-launch" xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M19 19H5V5h7V3H5a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
+                ></path>
+              </symbol>
+            </svg>
+          </li>
+          <li class="usa-identifier__required-links-item">
+            <a
+              href="https://www.gsa.gov/reference/reports/budget-performance"
+              class="usa-identifier__required-link usa-link"
+              >Performance reports</a
+            >
+            <svg class="usa-icon" aria-hidden="true" role="img">
+              <use xlink:href="#svg-launch"></use>
+              <symbol viewBox="0 0 24 24" id="svg-launch" xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M19 19H5V5h7V3H5a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
+                ></path>
+              </symbol>
+            </svg>
+          </li>
+          <li class="usa-identifier__required-links-item">
+            <a
+              href="https://www.gsa.gov/website-information/website-policies#privacy"
+              class="usa-identifier__required-link usa-link"
+              >Privacy policy</a
+            >
+            <svg class="usa-icon" aria-hidden="true" role="img">
+              <use xlink:href="#svg-launch"></use>
+              <symbol viewBox="0 0 24 24" id="svg-launch" xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M19 19H5V5h7V3H5a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
+                ></path>
+              </symbol>
+            </svg>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <section
+      class="usa-identifier__section usa-identifier__section--usagov"
+      aria-label="U.S. government information and services,,,"
+    >
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__usagov-description">
+          Looking for U.S. government information and services?
+        </div>
+        <a href="https://www.usa.gov/" class="usa-link">Visit USA.gov</a>
+        <svg class="usa-icon" aria-hidden="true" role="img">
+          <use xlink:href="#svg-launch"></use>
+          <symbol viewBox="0 0 24 24" id="svg-launch" xmlns="http://www.w3.org/2000/svg">
+            <path
+              d="M19 19H5V5h7V3H5a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
+            ></path>
+          </symbol>
+        </svg>
+      </div>
+    </section>
+  </div>
 </footer>
```

Comment:
possible to include instead of a lot of copy/paste?
```suggestion
{% include "snippets/identifier.html" %}
</footer>
```